lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/model-speed-v-mpg.adoc

```diff
@@ -0,0 +1,53 @@
+= Modeling Fuel Efficiency v. Speed
+++++
+<style>
+.studentAnswerMedium { min-width: 10em !important; }
+td { padding: 0 !important; }
+body.workbookpage .fitb { padding-top: 0.3rem !important; }
+</style>
+++++
+
+For this page, you'll need to have the @starter-file{alg2-fuel-efficiency} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@vspace{1ex}
+
+
+**In this exercise you'll have a chance to build and fit more models to the data, now armed with new tools that let you experiment with squares and square roots.** These tools are listed here (as well as in your contracts pages):
+
+@vspace{1ex}
+
+[cols="^1a,^1a", stripes="none", frame="none"]
+|===
+| @math{x^2}
+| @math{\sqrt x}
+
+| @show{(contract "num-sqr" '("Num") "Num" "Consumes a number, and produces the square of that number")}
+| @show{(contract "num-sqrt" '("Num") "Num" "Consumes a number, and produces the square root of that number")}
+|===
+
+@n In the Interactions Area, enter the following expressions, one after the other. Can you predict what the output will be? @show{(code '(sqr 16))}, @show{(code '(sqrt 4))}, @show{(code '(sqr (+ 10 6)))}, @show{(code '(sqrt (* 2 2)))}
```

Comment:
- I added fitb and solutions.
- Did you mean to have students squaring 16 so that some students would be able to do it in their head and others would not?

Reply:
Changed the numbers to make this easier.